nestkernel/kernel_manager.h

```diff
@@ -263,9 +264,18 @@ class KernelManager
    */
   void write_to_dump( const std::string& msg );
 
+  /**
+   * \defgroup Manager components in NEST kernel
+   *
+   * The managers are defined below in the order in which they need to be initialized.
```

Comment:
The order of initialization is defined by the initializer list in `kernel_manager.cpp`, so this statement is a bit misleading. I'd try to rephrase:
```suggestion
   * The list managers below should be kept in the same order in which they are
     initialized in `kernel_manager.cpp`.
```

Reply:
Members are initialized in order of declaration in the class definition, not in the order of the initializer list in the constructor. Compilers (should) warn about it if the order in the initializer list differs from order of declaration. The reason for this is that the destructor needs to destruct in reverse order of initialization and that is only reliably known from the declaration order; see section "Initialization order" in https://en.cppreference.com/w/cpp/language/constructor.